test/compflags/albrecht/chplenv/chplenv.precomp

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh
+
+# Pre-cleanup
+rm -f chplenv.compopts
+rm -f chplenv.good
+
+# Populate printchplenv variable
+printchplenv=$(../../../../util/printchplenv --simple)
+
+# Generate chplenv.compflags and chplenv.good
+compflags="--home=${CHPL_HOME}"
+for var in ${printchplenv}; do
+    echo ${var} >> chplenv.good
+    env=$(echo "${var}" | awk -F'=' '{print $1}')
+    value=$(echo "${var}" | awk -F'=' '{print $2}')
+    flag=$(echo "${env}" | sed 's/CHPL_//' | sed 's/_/-/g' | awk '{print tolower($0)}')
+    case $env in
+        "CHPL_TARGET_PLATFORM")
+            compflags="$compflags --$flag=$value"
+            ;;
+        "CHPL_TARGET_COMPILER")
+            compflags="$compflags --$flag=$value"
+            ;;
+        "CHPL_TARGET_ARCH")
+            compflags="$compflags --$flag=$value"
+            ;;
+        "CHPL_LOCALE_MODEL")
+            compflags="$compflags --$flag=$value"
+            ;;
+        "CHPL_COMM")
+            compflags="$compflags --$flag=$value"
+            ;;
+        "CHPL_COMM_SUBSTRATE")
+            compflags="$compflags --$flag=$value"
+            ;;
+        "CHPL_GASNET_SEGMENT")
+            compflags="$compflags --$flag=$value"
+            ;;
+        "CHPL_TASKS")
+            compflags="$compflags --$flag=$value"
+            ;;
+        "CHPL_LAUNCHER")
+            compflags="$compflags --$flag=$value"
+            ;;
+        "CHPL_TIMERS")
+            compflags="$compflags --$flag=$value"
+            ;;
+        "CHPL_MEM")
+            compflags="$compflags --$flag=$value"
+            ;;
+        "CHPL_MAKE")
+            compflags="$compflags --$flag=$value"
+            ;;
+        "CHPL_ATOMICS")
+            compflags="$compflags --$flag=$value"
+            ;;
+        "CHPL_NETWORK_ATOMICS")
+            compflags="$compflags --$flag=$value"
+            ;;
+        "CHPL_GMP")
+            compflags="$compflags --$flag=$value"
+            ;;
+        "CHPL_HWLOC")
+            compflags="$compflags --$flag=$value"
+            ;;
+        "CHPL_REGEXP")
+            compflags="$compflags --$flag=$value"
+            ;;
+        "CHPL_WIDE_POINTERS")
+            compflags="$compflags --$flag=$value"
+            ;;
+        "CHPL_AUX_FILESYS")
+            compflags="$compflags --$flag=$value"
+            ;;
+    esac
+done
+
+# Generate chplenv.good
+echo "CHPL_HOME=$CHPL_HOME" >> chplenv.good
+
+# Generate chplenv.compopts
+echo $compflags > chplenv.compopts
```

Comment:
I don't think we support dynamically created .compopts files. 

sub_test reads in the .compopts files before precomp is executed, so I don't think it will ever pick up the new .compopts. 

I checked with a hello.precomp of 

```bash
#!/bin/bash

echo "--fast" > hello.compopts
```

and --fast was never thrown to the compiler.